src/website/views/components/avatar.pug

```diff
@@ -0,0 +1,323 @@
+      <div class="a-avatar -xxl">AA</div>
+
+  #sizedIcons.a-tabs-panel
+    .-p3
+      .a-grid.-align-items--end
+        each size in sizes
+          div(class=`a-col -w${columnWidths[size]} -textCenter`)
+            div(class=`a-avatar a-icon -${size}`)
+              svg
+                use(xlink:href=`#icon-user`)
+            p.-text= size
+
+    :code(lang="html")
+      <div class="a-avatar a-icon -xs">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -sm">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -sm2">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -sm3">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -md">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -lg">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -xl">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+      <div class="a-avatar a-icon -xxl">
+        <svg>
+          <use xlink:href="#icon-user"></use>
+        </svg>
+      </div>
+  #sizedImages.a-tabs-panel
+    .-p3
+      .a-grid.-align-items--end
+        each size in sizes
+          div(class=`a-col -w${columnWidths[size]} -textCenter`)
+            div(class=`a-avatar -${size}`)
+              img(src='../../assets/images/avatar.jpg')(alt="avatar")
+            p.-text= size
+    :code(lang="html")
+      <div class="a-avatar a-icon -xs">
+        <img src="path/to/image.jpg">
+      </div>
+      <div class="a-avatar a-icon -sm">
+        <img src="path/to/image.jpg">
+      </div>
+      <div class="a-avatar a-icon -sm2">
+        <img src="path/to/image.jpg">
+      </div>
+      <div class="a-avatar a-icon -sm3">
+        <img src="path/to/image.jpg">
+      </div>
+      <div class="a-avatar a-icon -md">
+        <img src="path/to/image.jpg">
+      </div>
+      <div class="a-avatar a-icon -lg">
+        <img src="path/to/image.jpg">
+      </div>
+      <div class="a-avatar a-icon -xl">
+        <img src="path/to/image.jpg">
+      </div>
+      <div class="a-avatar a-icon -xxl">
+        <img src="path/to/image.jpg">
+      </div>
+
+h3 Labeled avatars
+
+p.-text
+  | Avatars can be accompanied by a label. Include the avatar and the label in a div with a <code>a-avatarGroup</code>
+  | class. Labeled avatars still support type, size and color variations.
+.example.-mb3
+  .-p3.a-grid
+    each color in colors
+      .-mb1.a-col.-w3
+        .a-avatarGroup
+          div(class=`a-avatar -${color}`) L
+          = `Label ${color}`
+  :code(lang="html")
+    <div class="a-avatarGroup">
```

Comment:
Need to change all instances of `a-avatarGroup` to `m-avatarGroup ` as it is a molecule - just like button groups.